net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,199 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access (pd doesn't yet work with portals)
+  - --filesystem=host
```

Comment:
can we limit to home at least?

Reply:
How would I do that?

Reply:
you are mistaken. 

This is not for device access.

Reply:
Got it. So, how do I specify just access to the home folder?

Reply:
does you code call it directly? Then it should instead use the portal. If it doesn't call it, then there is no point.

Reply:
Thanks, unfortunately pd and it's forks doesn't work with portals yet, so access to the home dir a must at this point in time. I changed this to --filesystem=home.